apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -221,59 +221,60 @@ type EBPFFlowFilter struct {
 	// Examples: `10.10.10.0/24` or `100:100:100:100::/64`
 	CIDR string `json:"cidr,omitempty"`
 
-	// `action` defines the action to perform on the flows that match the filter.
+	// `action` defines the action to perform on the flows that match the filter. The available options are `Accept` and `Reject`.
```

Comment:
agent default to `Accept` I don't think we need to default to Accept in the API but should be say we have a default ?

Reply:
I thought it was actually required.. Well maybe I'm for setting an explicit default to Accept then, so it's clearly set in API and not handed over to agent's own default?

Reply:
done